lib/assessment.rb

```diff
@@ -13,28 +13,29 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum..inject{|sum,x| sum + x }
+    # reduce(&:+) i think this works too??
```

Comment:
Correct! (minus the extra dot)
`Inject` is actually an alias to `reduce` - they're the same.